### PR TITLE
[DOCS-6741] Alfresco Intelligence Services 1.4.4

### PR DESCRIPTION
**Pull request by anxumalo**
Summary of changes:

- For AIS 1.3.x: updated links to point to Content Services 7.1 docs.
- For AIS 1.4.4: updated supported AIS version in Content Services 7.2 Supported platforms page, and changed references to Digital Workspace 2.8, and Transform Service 1.5. The previous versions should be backward compatible.

### CODE REVIEW

File: intelligence-services/latest/config/index.md [hunk 1]
```diff
@@ -111,14 +111,15 @@ A number of environment variables allow you to specify the configuration options
...
     ```yaml
-  digital-workspace:
-    image: quay.io/alfresco/alfresco-digital-workspace:2.4.0-adw
-    environment:
-      BASE_PATH: ./
-      APP_CONFIG_PLUGIN_AI_SERVICE: "true"
-    volumes:
-      - ./ai-view-extension.json:/usr/share/nginx/html/assets/plugins/ai-view-extension.json
+    digital-workspace:
+        image: quay.io/alfresco/alfresco-digital-workspace:2.4.2
```
Comment (alandavis) [inline intelligence-services/latest/config/index.md:115]: This version looks very old. It is from 7 months ago. The https://github.com/Alfresco/acs-deployment/blob/master/docker-compose/docker-compose.yml#L149 currently has the ACS 7.2.0 compatible values and is 2.7.0. There is also now a 2.8.0, but I'm not the right person to know what version should be here.

There is a second reference to 2.4.2 in this PR, if you wish to change this.

File: intelligence-services/1.3/index.md [hunk 2]
```diff
@@ -22,6 +22,6 @@ Other features introduced in previous versions include:
 * Enrichment of content metadata (v1.0)
 * Easier access to insights from unstructured content (v1.0)
 
-> **Important:** The Intelligence Services module can be applied to Content Services 6.2 or later. See [Supported platforms]({% link intelligence-services/1.3/support/index.md %}) for more.
+> **Important:** The Intelligence Services module can be applied to Content Services 7.0. See [Supported platforms]({% link intelligence-services/1.3/support/index.md %}) for more.
```
Comment (alandavis) [inline intelligence-services/1.3/index.md:25]: Later on you reference 7.2 rather than 7.0.

**Review by alandavis — CHANGES_REQUESTED** [review]
Please see my comment about the ADW version.

I note that you have increased the base ACS and ATS version. This probably did not need to happen, as they should be backward compatible, but is probably a good idea to have updated them anyway. I would suggest using ACS 7.2.

Once changes have been made, I don't think there is any need for a further review from me.

**Review by oliveralfresco2018 — APPROVED** [review]
LGTM, built and it worked too.

File: intelligence-services/latest/config/index.md (hunk 1 above)
Comment (anxumalo) [reply intelligence-services/latest/config/index.md]: Switched to ADW 2.8

**Pull request merged**